source/unified-test-format/unified-test-format.rst

```diff
@@ -462,6 +478,49 @@ The structure of this object is as follows:
 
     Test files SHOULD NOT use this option unless one or more command monitoring
     events are specified in `observeEvents <entity_client_observeEvents_>`_.
+    
+  - ``storeEventsAsEntities``: Optional map of entity names to an array of
+    event names.
+    
+    For each entity name, the test runner MUST create the respective entity
```

Comment:
AFAICT this field does not reuse/rely on information specified in the `observeEvents` and `ignoreCommandMonitoringEvents` however this is not clear on first reading this section. I suggest rephrasing this as:

For each entity name, the test runner MUST:

1. create the respective entity with a type of "event list", as described in `Supported Entity Types`_. If the entity already exists (such as from a previous ``storeEventsAsEntities`` declaration from another client), the test runner MUST raise an error.
2. set up an event subscriber for each event named. The event subscriber MUST serialize the events it receives into a document, using the documented properties of the event as field names, and append the document to the event list created in the previous step.
...

Reply:
I added a note stating that storeEventsAsEntities is independent from observeEvents and ignoreCommandMonitoringEvents, does that address your concern?

Reply:
Yes, that does it. Resolving.